builtin/builtin.go

```diff
@@ -868,6 +868,40 @@ var Builtins = []*Function{
 			return anyType, fmt.Errorf("cannot transform %s from pairs", args[0])
 		},
 	},
+	{
+		Name: "reverse",
+		Func: func(args ...any) (any, error) {
+			if len(args) != 1 {
+				return nil, fmt.Errorf("invalid number of arguments (expected 1, got %d)", len(args))
+			}
+
+			v := reflect.ValueOf(args[0])
+			if v.Kind() != reflect.Slice && v.Kind() != reflect.Array {
+				return nil, fmt.Errorf("cannot reverse %s", v.Kind())
+			}
+
+			size := v.Len()
+			arr := make([]any, size)
+
+			for i := 0; i < size; i++ {
+				arr[i] = v.Index(size - i - 1).Interface()
+			}
+
+			return arr, nil
+
+		},
+		Validate: func(args []reflect.Type) (reflect.Type, error) {
+			if len(args) != 1 {
+				return anyType, fmt.Errorf("invalid number of arguments (expected 1, got %d)", len(args))
+			}
+			switch kind(args[0]) {
+			case reflect.Interface, reflect.Slice, reflect.Array:
+			default:
+				return anyType, fmt.Errorf("cannot reverse %s", args[0])
```

Comment:
Let's move this return inside `case`.

Reply:
What you mean? you still will need return from the function

Reply:
Nope.

```go
// You can edit this code!
// Click here and start typing.
package main

import "fmt"

func main() {
	fmt.Println(foo(42))
}

func foo(i int) bool {
	switch i {
	case 42:
		return true
	default:
		return false
	}
}
```
https://go.dev/play/p/YtfTFf9GjZ9